UoaWDCC/AEYB
# Aeyb/#121 env var

**Closes:** #121

**What was done:**
- Replaced all `config.json` references, to `.env` references

**How to test:**

-   [ ] Check that everything works as expected. (e.g. login, creating meetings, viewing meetings, etc.)
